tests/fixtures/db/__init__.py

```diff
@@ -18,6 +18,17 @@ def engine_uri_env(engine_uri):
         monkeypatch.undo()
 
 
+@pytest.fixture
+def un_engine_uri():
+    """When we want to test functionality that doesn't use the current DB URI"""
+    monkeypatch = MonkeyPatch()
```

Comment:
```suggestion
def un_engine_uri(monkeypatch):
    """When we want to test functionality that doesn't use the current DB URI"""
```

Reply:
Refactored the engine_uri function and this one to use the monkeypatch fixture, and use context managers instead! Thanks for the suggestion earlier I use context managers a lot but I haven't written many! I appreciate the code review